src/index.js

```diff
@@ -0,0 +1,154 @@
+const raiseTemp = () => {
+  const temperature = document.getElementById('display-temp');
+  temperature.innerText++;
+  changeTempColor();
+  changeLandscape();
+};
+
+const lowerTemp = () => {
+  const temperature = document.getElementById('display-temp');
+  temperature.innerText--;
+  changeTempColor();
+  changeLandscape();
+};
+
+const changeTempColor = () => {
+  const temperature = document.getElementById('display-temp');
+  if (temperature.innerText >= 80) {
+    temperature.style.color = 'lightcoral';
+  } else if (temperature.innerText >= 70 && temperature.innerText <= 79) {
+    temperature.style.color = 'blueviolet';
+  } else if (temperature.innerText >= 60 && temperature.innerText <= 69) {
+    temperature.style.color = 'peru';
+  } else if (temperature.innerText >= 50 && temperature.innerText <= 59) {
+    temperature.style.color = 'green';
+  } else {
+    temperature.style.color = 'teal';
+  }
+};
+
+const changeLandscape = () => {
+  const temperature = document.getElementById('display-temp'); // should I be using const or let - since it changes?
+  let landscape = document.getElementById('landscape');
+  if (temperature.innerText >= 80) {
+    landscape.textContent = '🌵_🐍_🌴_🦂_🌴_🌵🌵_🌴_🐍_🏜_🌴_🦂';
+  } else if (temperature.innerText >= 70 && temperature.innerText <= 79) {
+    landscape.textContent = '🌸🌿🌼🍀🌷🌻🌿🍀🌱🍀🌻🌷';
+  } else if (temperature.innerText >= 60 && temperature.innerText <= 69) {
+    landscape.textContent = '🌾🌾🪴🍃🪴🪨🪴🛤🪴🌾🌾🌾🪴🍃';
+  } else {
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+
+const updateCityName = () => {
+  let currentCityName = document.getElementById('city-name-display'); // where city name is currently displayed
+  let cityNameUserInput = document.getElementById('city-name').value; // should give value of input
+  currentCityName.textContent = cityNameUserInput;
+};
+
+/*
+Sorry about lack of readability - was supposed to put into two functions
```

Comment:
No worries! It works well. Yes, it would be more readable to split the location and weather calls to separate functions.